src/pages/management/configuration-api/v3.6/index.mdx

```diff
@@ -285,12 +285,23 @@ curl -X POST \
             "priority": "normal"
           }
         ],
-        "work_scheduler": {
-          "monday": {
-            "start": "08:30",
-            "end": "12:30"
-          }
-        },
+       "work_scheduler": {
```

Comment:
@gluszczyk, I was not sure, but I also changed it in v3.6. Please tell me if that was a bad decision so I will remove it 😉

Reply:
good decision,
but pls also add a third entry in both versions 😉 
```
            {
                "day": "friday",
                "start": "08:00",
                "end": "16:00",
                "enabled": true
            }
```

Reply:
Done :) If everything looks OK, please let me know, and I will publish the changes😊